addons/project/report/project_report.py

```diff
@@ -34,7 +34,7 @@ class ReportProjectTaskUser(models.Model):
         ('0', 'Low'),
         ('1', 'High')
         ], readonly=True, string="Priority")
-    state = fields.Selection([
+    kanban_state = fields.Selection([
```

Comment:
IMO, no need, the kanban state will be removed in that PR: #107593 🙂

Reply:
in fact the field is not longer in the search view, so it is okay to revert this change. 🙂